argus/recipes/cloud/windows.py

```diff
@@ -327,45 +334,36 @@ def prepare_cbinit_config(self, service_type):
         # Append service IP as a config option.
         address = self.pattern.format(util.get_local_ip())
         self._cbinit_conf.set_conf_value(name=self.config_entry,
-                                         value=address)
+                                         value=address,
+                                         section=self.config_group)
 
 
 class CloudbaseinitEC2Recipe(CloudbaseinitMockServiceRecipe):
     """Recipe for EC2 metadata service mocking."""
 
-    config_entry = "ec2_metadata_base_url"
+    config_group = "ec2"
```

Comment:
We could use constant instead of magic strings like `util.EC2_SERVICE` https://github.com/cloudbase/cloudbase-init-ci/blob/master/argus/util.py#L48-L53